### DISCUSSION TITLE
Calibrate points in an plane of the house

### DISCUSSION THREAD

**Discussion by raider206** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#222] [Q&A] [answered]
Hello!

I have the Xiaomi Vacuum (the first model I think) and I am trying to configure this vacuum card. 

This is the last map it did from my house:
![image](https://user-images.githubusercontent.com/73711745/143432810-72d7b78c-f453-4635-9ab6-b83960c979ec.png)

This is the "plane" of my house:
![image](https://user-images.githubusercontent.com/73711745/143432980-b7582552-150c-43c8-83cf-a2faa69b3dd8.png)

My intention is to correspond the vacuum point on the image. As fas as I read I have to read the vacuum points and to correspond them on the image. I have to put them on the "calibration points". Is it correct? I want to mark the different rooms on this image to have always the same plane. 

My vacuum only save the last cleaning map but it can change a little when it comes back to the base or if I clean it and I return to the base. I think that the "vacuum points" not will be exactly the same the next time it will clean. It they change, the calibration point will work fine?? Am I doing something wrong ?

I've tried to explain it, but if you don't understand something, tell me and I'll try to explain better.

Regards

**Comment by PiotrMachowski** [comment, marked as answer]
Yes, this is what needs to be done to use an image instead of a camera.

You have to adjust calibration points every time a map changes. I assume you use gen1 vacuum? In this case I would suggest to just use a camera

**Reply by raider206** [reply]
Yes, its the gen1 vacuum. 

Yo its better to use the map that generates the camera instead of the image.

Thank you!
